hal/src/main/java/edu/wpi/first/hal/DIOJNI.java

```diff
@@ -178,4 +178,7 @@ public class DIOJNI extends JNIWrapper {
    * @see "HAL_SetDigitalPWMOutputChannel"
    */
   public static native void setDigitalPWMOutputChannel(int pwmGenerator, int channel);
+
+  /** Utility class. */
+  DIOJNI() {}
```

Comment:
Why is this one package-private instead of private?

Reply:
DIOJNI is inherited by some other JNIs.